flow/embedded_views.h

```diff
@@ -142,6 +142,7 @@ class MutatorsStack {
   // Returns an iterator pointing to the bottom of the stack.
   const std::vector<std::shared_ptr<Mutator>>::const_reverse_iterator Bottom()
       const;
+  bool empty() const { return vector_.empty(); }
```

Comment:
`is_empty` might be clearer. This makes it sound like it empties the mutator stack.